pkg/util/envvar/envvars.go

```diff
@@ -26,3 +27,10 @@ func MergeWithOverride(existing, desired []corev1.EnvVar) []corev1.EnvVar {
 	})
 	return mergedEnv
 }
+
+func GetEnvOrDefault(envVar, defaultValue string) string {
+	if val := os.Getenv(envVar); val != "" {
+		return val
+	}
+	return defaultValue
```

Comment:
[nit] Instead on relying on the value returned to be empty, why not using [LookupEnv](https://golang.org/pkg/os/#LookupEnv)?

Reply:
good catch, will switch to this